### PR TITLE
feat: added configurable keybindings to alter config values at run time

### PR DESCRIPTION
**Pull request by ramirezmike**
I added some config bindings to flip/increment/decrement a few config settings in order to setup something like "Alt+I" toggles display of the appbar. I tend to setup i3 on my linux box this way and kept wanting to hide the bar when not in use but also I found in windows sometimes I do want to see the task bar, like for altering stuff in the tray icons window, so this also let me show/hide the task bar without editing my config file. Figured it can be used for other things like adjusting sizes of things too. Let me know what you think.

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -104,6 +104,39 @@ impl Config {
...
+    }
+
+    fn alter_numerical_field(self: &mut Self, field: &str, value: i32) {
+        match field {
+            "app_bar_height" => self.app_bar_height += value,
+            "app_bar_bg" => self.app_bar_bg += value,
+            "app_bar_font_size" => self.app_bar_font_size += value,
+            "margin" => self.margin += value,
+            "padding" => self.padding += value,
+            _ => debug!("Attempt to alter unknown field: {} by {}", field, value)
```
Comment (TimUntersberger) [inline src/config.rs:123]: I feel like this should be more an `error!` than a `debug!`.
```suggestion
            _ => error!("Attempt to alter unknown field: {} by {}", field, value)
```

File: src/config.rs [hunk 2]
```diff
@@ -104,6 +104,39 @@ impl Config {
...
+        match field {
+            "use_border" => self.use_border = !self.use_border,
+            "work_mode" => self.work_mode = !self.work_mode,
+            "light_theme" => self.light_theme = !self.light_theme,
+            "multi_monitor" => self.multi_monitor = !self.multi_monitor,
+            "launch_on_startup" => self.launch_on_startup = !self.launch_on_startup,
+            "remove_title_bar" => self.remove_title_bar = !self.remove_title_bar,
+            "remove_task_bar" => self.remove_task_bar = !self.remove_task_bar,
+            "display_app_bar" => self.display_app_bar = !self.display_app_bar,
+            _ => debug!("Attempt to toggle unknown field: {}", field)
```
Comment (TimUntersberger) [inline src/config.rs:137]: Same thing here.
```suggestion
            _ => error!("Attempt to toggle unknown field: {}", field)
```

**Comment by ramirezmike** [comment]
Agreed, updated. Also, I removed work_mode and multi_monitor as work_mode has a dedicated keybinding and I think multi_monitor might require additional work in the hot reload to work correctly.

**Comment by TimUntersberger** [comment]
> I think multi_monitor might require additional work

Nice catch!

**LGTM**

**Pull request merged**